### PR TITLE
Add ERA5-forced ACCESS-OM2 runs

### LINKED ISSUE #127: [DATA REQUEST] Add ACCESS-OM2 runs using ERA5
# Description of the data product

ACCESS-OM2 Runs using ERA5 as a data atmosphere to force the model instead of JRA55

There are 8 runs - for each combination of IAF/RYF, ERA5/JRA55, 0.25/1 deg

# Location of the data product on Gadi

Data location in /g/data/ik11/outputs/ (see https://forum.access-hive.org.au/t/era-5-forced-access-om2-simulations/1103)

# Checklist

Add a "x" between the brackets to all that apply
- [x] This data product is stable (unlikely to change substantially or move)
- [x] This data product is of use to the broader community
- [x] This data product is documented: (see links in https://forum.access-hive.org.au/t/era-5-forced-access-om2-simulations/1103)
- [x] This data product is licensed under 'Acknoledge COSIMA'
- [x] Those who want to access this data can be added to the project that houses it

### PR DESCRIPTION
**Pull request by dougiesquire**
Adds the runs described in this ACCESS-Hive Forum post to the catalog: https://forum.access-hive.org.au/t/era-5-forced-access-om2-simulations/1103

Closes #127

### CODE REVIEW

**Comment by dougiesquire** [comment]
Some additional fields need to be added to the `metadata.yaml` files for these runs. I've created new versions of these files with the additional fields here: `/scratch/ik11/ds0092`. @rmholmes would you mind taking a look at these and copying them to the corresponding run directories if you're happy with them?

If you think it's worthwhile, it could be helpful to add some additional info to the descriptions fields to clarify:
- `1deg_jra55_iaf_era5comparison`: how is this different than the `1deg_jra55_iaf_omip2_cycle*` runs?
- `1deg_jra55v14_ryf`: how is this different than the `1deg_jra55_ryf9091_gadi` run? Just the JRA55 version?
- `025deg_jra55_iaf_era5comparison`: how is this different than the `025deg_jra55_iaf_omip2_cycle*` runs?
- `025deg_jra55_ryf_era5comparison`: how is this different than the `025deg_jra55_ryf9091_gadi` run? Just the JRA55 version?

**Comment by dougiesquire** [comment]
@anton-seaice would you mind casting your eyes on this?

File: config/access-om2.yaml
Comment (anton-seaice) [inline config/access-om2.yaml:1]: I tried to run a test copy of 'build_all.sh' to output to scratch, and it gave the error the 'name' property is missing in the first added file. I didn't check the other files or for other missing properties.

I remember we talked about versioning, but would it be clearer if the metadata.yaml has the github version in it?

**Review by anton-seaice — CHANGES_REQUESTED** [review]

File: config/access-om2.yaml [hunk 1]
```diff
@@ -7,12 +7,22 @@ sources:
   - metadata_yaml: /g/data/ik11/outputs/access-om2/1deg_jra55_ryf9091_gadi/metadata.yaml
     path:
       - /g/data/ik11/outputs/access-om2/1deg_jra55_ryf9091_gadi
-      
-    
+
+
+  - metadata_yaml: /g/data/ik11/outputs/access-om2/1deg_jra55v14_ryf/metadata.yaml
```
[the remark this answers is not in the document]
Comment (dougiesquire) [reply config/access-om2.yaml:12]: Yeah, all of the `metadata.yaml` files in `/g/data/ik11/outputs` are missing a number of fields at the moment. I've created versions of the files with the necessary fields added here `/scratch/ik11/ds0092` and ask Ryan to check these before copying them across to the `/g/data/ik11/outputs`. See this comment: https://github.com/ACCESS-NRI/access-nri-intake-catalog/pull/130#issuecomment-1766117003

File: config/access-om2.yaml
Comment (dougiesquire) [reply config/access-om2.yaml:1]: > but would it be clearer if the metadata.yaml has the github version in it?

The github version of what?

File: config/access-om2.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (anton-seaice) [reply config/access-om2.yaml:12]: Oh ofcourse :)

File: config/access-om2.yaml
Comment (anton-seaice) [reply config/access-om2.yaml:1]: This line

`url: https://github.com/rmholmes/1deg_era5_iaf/tree/ryan_testing`

doesn't specify a unique version - just the repo, which could be changed after the run etc

**Comment by anton-seaice** [comment]
I think this is not accurate for the ERA runs
```
1 degree ACCESS-OM2 global model configuration under '
                    'the RYF9091 Repeat\n'
                    'Year Forcing strategy outlined by Stewart et al. '
                    '(2020), \n'
                    'https://doi.org/10.1016/j.ocemod.2019.101557.
```

This document is about JRA55

File: config/access-om2.yaml
Comment (dougiesquire) [reply config/access-om2.yaml:1]: I see. Yeah, including the commit hash is definitely a good idea where it makes sense to do so. In many cases, the configuration changes throughout the model run, so there isn't one hash. My approach has been to leave this up to the person who ran the model

**Comment by dougiesquire** [comment]
> I think this (from the description) is not accurate for the ERA runs
> 
> ```
> 1 degree ACCESS-OM2 global model configuration under '
>                     'the RYF9091 Repeat\n'
>                     'Year Forcing strategy outlined by Stewart et al. '
>                     '(2020), \n'
>                     'https://doi.org/10.1016/j.ocemod.2019.101557.
> ```
> 
> The document in the DOI is about JRA55

Hopefully @rmholmes can comment/improve

**Comment by rmholmes** [comment]
Sorry for the slowness on this.

Thanks @dougiesquire for providing the updated meta data. I've added a few updates on top of yours and copied them to the appropriate directories in `ik11`. If you want to see what I changed, see the last two commits at https://github.com/rmholmes/era5_metas/commits/main. I think they should be all good now.

**Comment by dougiesquire** [comment]
Great - thanks heaps @rmholmes!

**Comment by anton-seaice** [comment]
Based on the updated metadata.yaml files for the 8 relevant runs being updated and moved in ik11, I managed to build datastores (using bin/build_all.sh) for new added model runs.

Log files attached

[v127_OM2_ERA5.tar.gz](https://github.com/ACCESS-NRI/access-nri-intake-catalog/files/13232869/v127_OM2_ERA5.tar.gz)
[conda_env.yaml.txt](https://github.com/ACCESS-NRI/access-nri-intake-catalog/files/13232880/conda_env.yaml.txt)
[build_all.sh_as.o99597320.txt](https://github.com/ACCESS-NRI/access-nri-intake-catalog/files/13232881/build_all.sh_as.o99597320.txt)

**Review by anton-seaice — APPROVED** [review]

**Pull request merged**